lib/timezone/lookup/geonames.rb

```diff
@@ -25,9 +25,7 @@ def lookup(lat, long)
 
         data = JSON.parse(response.body)
 
-        return data['timezoneId'] if data['timezoneId']
-
-        return unless data['status']
+        return get_timezone_id(data) unless data['status']
```

Comment:
Inlining it won't work with rubocop - too mych complexity in the lookup method. That's the reason I had moved so much into get_timezone_id method. But I've returned old behaviour with data['status'].